ElectronClient/app/app.js

```diff
@@ -850,6 +865,16 @@ class Application extends BaseApplication {
 							name: 'toggleSidebar',
 						});
 					},
+				},
+				{
```

Comment:
They should be on the same line: `}, {`
It's strange that the linter didn't complain.

Reply:
This should be good now :+1: